apps/chatter/classes.py

```diff
@@ -0,0 +1,14 @@
+class Entry(object):
+    def __init__(self, text, timestamp, author, room):
+        self.text = text
+        self.timestamp = timestamp
+        self.author = author
+        self.room = room
```

Comment:
What is the purpose of having an Entry model and an Entry class?

Reply:
Good question! I was experimenting with some of the examples on serializers trying to figure out what made the most sense. Removing!